### PR TITLE
libigl: add run_tests.sh

### PR DESCRIPTION
**Pull request by vitaliset**
Adds `run_tests.sh` to the libigl project.

`run_tests.sh` is used as part of Chronos with cached builds: https://github.com/google/oss-fuzz/tree/master/infra/experimental/chronos#check-tests

Output of ./infra/experimental/chronos/check_tests.sh libigl c++:
```
Test project /src/libigl/build-dir
        Start   1: AABB: find_2d
        Start   2: AABB: find_3d
        Start   3: AABB: insert
        Start   4: AABB: dynamic
        Start   5: MshLoader
        Start   6: MshSaver
        Start   7: accumarray: matlab_help
        Start   8: accumarray: scalar
  1/214 Test   #3: AABB: insert ............................................   Passed    0.11 sec
        Start   9: adjacency_list: simple
  2/214 Test   #1: AABB: find_2d ...........................................   Passed    0.46 sec
        Start  10: avg_edge_length: cube
  3/214 Test   #2: AABB: find_3d ...........................................   Passed    0.14 sec
        Start  11: barycentric_interpolation: two-triangles
  4/214 Test   #8: accumarray: scalar ......................................   Passed    0.13 sec
        Start  12: bbw: decimated_knight
  5/214 Test   #7: accumarray: matlab_help .................................   Passed    0.14 sec
        Start  13: bezier: ease
  6/214 Test   #9: adjacency_list: simple ..................................   Passed    0.12 sec
        Start  14: blkdiag: 3-matrices
  7/214 Test  #13: bezier: ease ............................................   Passed    0.10 sec
        Start  15: boundary_facets: single_tet_volume
  8/214 Test  #11: barycentric_interpolation: two-triangles ................   Passed    0.13 sec
        Start  16: boundary_facets: single_tet
  9/214 Test   #5: MshLoader ...............................................   Passed    0.27 sec
        Start  17: boundary_facets: single_cube
 10/214 Test  #10: avg_edge_length: cube ...................................   Passed    0.16 sec
        Start  18: boundary_facets: non-manifold
 11/214 Test  #14: blkdiag: 3-matrices .....................................   Passed    0.10 sec
        Start  19: boundary_loop: cube
 12/214 Test  #17: boundary_facets: single_cube ............................   Passed    0.10 sec
        Start  20: boundary_loop: bunny
 13/214 Test  #18: boundary_facets: non-manifold ...........................   Passed    0.11 sec
        Start  21: cat: rows
 14/214 Test  #15: boundary_facets: single_tet_volume ......................   Passed    0.15 sec
        Start  22: cat: cols
 15/214 Test  #16: boundary_facets: single_tet .............................   Passed    0.15 sec
        Start  23: centroid: 
 16/214 Test   #6: MshSaver ................................................   Passed    0.45 sec
        Start  24: circulation: single_edge
 17/214 Test  #19: boundary_loop: cube .....................................   Passed    0.15 sec
        Start  25: circumradius: equilateral-triangle
 18/214 Test  #22: cat: cols ...............................................   Passed    0.09 sec
        Start  26: circumradius: right-triangle
 19/214 Test  #23: centroid:  ..............................................   Passed    0.11 sec
        Start  27: circumradius: obtuse-triangle
 20/214 Test  #24: circulation: single_edge ................................   Passed    0.08 sec
        Start  28: circumradius: equilateral-tetrahedra
 21/214 Test  #21: cat: rows ...............................................   Passed    0.20 sec
        Start  29: circumradius: right-tetrahedra
 22/214 Test  #26: circumradius: right-triangle ............................   Passed    0.11 sec
        Start  30: circumradius: obtuse-tetrahedra
 23/214 Test  #25: circumradius: equilateral-triangle ......................   Passed    0.13 sec
        Start  31: cotmatrix: poly
 24/214 Test  #27: circumradius: obtuse-triangle ...........................   Passed    0.12 sec
        Start  32: cotmatrix: constant_in_null_space
 25/214 Test  #28: circumradius: equilateral-tetrahedra ....................   Passed    0.14 sec
        Start  33: cotmatrix: cube
 26/214 Test  #29: circumradius: right-tetrahedra ..........................   Passed    0.13 sec
        Start  34: cotmatrix: tetrahedron
 27/214 Test  #30: circumradius: obtuse-tetrahedra .........................   Passed    0.19 sec
        Start  35: cotmatrix_entries: simple
 28/214 Test  #34: cotmatrix: tetrahedron ..................................   Passed    0.09 sec
        Start  36: cotmatrix_entries: intrinsic
 29/214 Test  #33: cotmatrix: cube .........................................   Passed    0.17 sec
        Start  37: cotmatrix_intrinsic: periodic
 30/214 Test  #35: cotmatrix_entries: simple ...............................   Passed    0.11 sec
        Start  38: cotmatrix_intrinsic: manifold_meshes
 31/214 Test  #36: cotmatrix_entries: intrinsic ............................   Passed    0.10 sec
        Start  39: cr_vector_curvature_correction: cube
 32/214 Test  #37: cotmatrix_intrinsic: periodic ...........................   Passed    0.11 sec
        Start  40: cr_vector_curvature_correction: annulus
 33/214 Test  #39: cr_vector_curvature_correction: cube ....................   Passed    0.14 sec
        Start  41: cr_vector_curvature_correction: hemisphere
 34/214 Test  #41: cr_vector_curvature_correction: hemisphere ..............   Passed    0.54 sec
        Start  42: cr_vector_laplacian: cube
 35/214 Test  #42: cr_vector_laplacian: cube ...............................   Passed    0.15 sec
        Start  43: cumprod: col_factorial
 36/214 Test  #43: cumprod: col_factorial ..................................   Passed    0.12 sec
        Start  44: cumprod: row_factorial
 37/214 Test  #44: cumprod: row_factorial ..................................   Passed    0.12 sec
        Start  45: cumsum: col
 38/214 Test  #20: boundary_loop: bunny ....................................   Passed    1.74 sec
        Start  46: cumsum: row
 39/214 Test  #45: cumsum: col .............................................   Passed    0.13 sec
        Start  47: curved_hessian_energy: cube
 40/214 Test  #46: cumsum: row .............................................   Passed    0.11 sec
        Start  48: curved_hessian_energy: annulus
 41/214 Test  #47: curved_hessian_energy: cube .............................   Passed    0.14 sec
        Start  49: cut_mesh: seperate mesh
 42/214 Test  #49: cut_mesh: seperate mesh .................................   Passed    0.11 sec
        Start  50: cut_mesh: single edge
 43/214 Test  #50: cut_mesh: single edge ...................................   Passed    0.26 sec
        Start  51: cut_mesh: two triangles
 44/214 Test  #48: curved_hessian_energy: annulus ..........................   Passed    0.46 sec
        Start  52: cut_to_disk: simple_tet
 45/214 Test  #52: cut_to_disk: simple_tet .................................   Passed    0.14 sec
        Start  53: cut_to_disk: two_disconnected_tet
 46/214 Test  #51: cut_mesh: two triangles .................................   Passed    0.18 sec
        Start  54: cut_to_disk: simple_square
 47/214 Test  #54: cut_to_disk: simple_square ..............................   Passed    0.08 sec
        Start  55: cut_to_disk: torus
 48/214 Test  #53: cut_to_disk: two_disconnected_tet .......................   Passed    0.21 sec
        Start  56: cut_to_disk: cube
 49/214 Test  #55: cut_to_disk: torus ......................................   Passed    0.14 sec
        Start  57: cut_to_disk: annulus
 50/214 Test  #56: cut_to_disk: cube .......................................   Passed    0.18 sec
        Start  58: decimate: hemisphere
 51/214 Test  #58: decimate: hemisphere ....................................   Passed    0.29 sec
        Start  59: decimate: closed
 52/214 Test  #59: decimate: closed ........................................   Passed    0.32 sec
        Start  60: delaunay_triangulation_issue_521
 53/214 Test  #60: delaunay_triangulation_issue_521 ........................   Passed    0.08 sec
        Start  61: dijkstra: cube
 54/214 Test  #61: dijkstra: cube ..........................................   Passed    0.09 sec
        Start  62: dijkstra: discrete distances
 55/214 Test  #57: cut_to_disk: annulus ....................................   Passed    0.99 sec
        Start  63: direct_delta_mush: cube
 56/214 Test  #62: dijkstra: discrete distances ............................   Passed    0.10 sec
        Start  64: dirname: examples
 57/214 Test  #64: dirname: examples .......................................   Passed    0.10 sec
        Start  65: doublearea: VF_vs_ABC
 58/214 Test  #63: direct_delta_mush: cube .................................   Passed    0.27 sec
        Start  66: ears: grid
 59/214 Test  #66: ears: grid ..............................................   Passed    0.15 sec
        Start  67: ears: two-boundary
 60/214 Test  #32: cotmatrix: constant_in_null_space .......................   Passed    4.15 sec
        Start  68: edge_exists_near: tet
 61/214 Test  #68: edge_exists_near: tet ...................................   Passed    0.09 sec
        Start  69: edge_flaps: verify
 62/214 Test  #38: cotmatrix_intrinsic: manifold_meshes ....................   Passed    5.39 sec
        Start  70: edge_lengths: cube
 63/214 Test  #70: edge_lengths: cube ......................................   Passed    0.11 sec
        Start  71: edges_to_path: simple
 64/214 Test  #71: edges_to_path: simple ...................................   Passed    0.12 sec
        Start  72: euler_characteristic: cube
 65/214 Test  #72: euler_characteristic: cube ..............................   Passed    0.08 sec
        Start  73: euler_characteristic: triangle
 66/214 Test  #73: euler_characteristic: triangle ..........................   Passed    0.10 sec
        Start  74: euler_characteristic: square
 67/214 Test  #74: euler_characteristic: square ............................   Passed    0.11 sec
        Start  75: euler_characteristic: torus
 68/214 Test  #75: euler_characteristic: torus .............................   Passed    0.13 sec
        Start  76: exact_geodesic: square
 69/214 Test  #76: exact_geodesic: square ..................................   Passed    0.15 sec
        Start  77: facet_components: two_triangles
 70/214 Test  #77: facet_components: two_triangles .........................   Passed    0.17 sec
        Start  78: facet_components: truck
 71/214 Test  #78: facet_components: truck .................................   Passed    0.29 sec
        Start  79: fast_winding_number: one_point_cloud
 72/214 Test  #79: fast_winding_number: one_point_cloud ....................   Passed    0.09 sec
        Start  80: fast_winding_number: meshes
 73/214 Test  #65: doublearea: VF_vs_ABC ...................................   Passed    3.84 sec
        Start  81: fit_cubic_bezier: hemicircle
 74/214 Test  #81: fit_cubic_bezier: hemicircle ............................   Passed    0.46 sec
        Start  82: grad: laplace_grid
 75/214 Test  #82: grad: laplace_grid ......................................   Passed    0.28 sec
        Start  83: grad_intrinsic: laplace_grid
 76/214 Test  #83: grad_intrinsic: laplace_grid ............................   Passed    0.20 sec
        Start  84: grid: 3d
 77/214 Test  #84: grid: 3d ................................................   Passed    0.36 sec
        Start  85: grid: 2d
 78/214 Test  #85: grid: 2d ................................................   Passed    0.42 sec
        Start  86: guess_extension: all_meshes
 79/214 Test  #86: guess_extension: all_meshes .............................   Passed    0.29 sec
        Start  87: heat_geodesic: upsampled cube
 80/214 Test  #87: heat_geodesic: upsampled cube ...........................   Passed    0.45 sec
        Start  88: icosahedron: simple
 81/214 Test  #88: icosahedron: simple .....................................   Passed    0.19 sec
        Start  89: internal_angles: 1e-7
 82/214 Test  #89: internal_angles: 1e-7 ...................................   Passed    0.24 sec
        Start  90: intersection_blocking_collapse_edge_callbacks: simple
 83/214 Test  #90: intersection_blocking_collapse_edge_callbacks: simple ...   Passed    0.20 sec
        Start  91: intrinsic_delaunay_cotmatrix: skewed_grid
 84/214 Test  #91: intrinsic_delaunay_cotmatrix: skewed_grid ...............   Passed    0.32 sec
        Start  92: intrinsic_delaunay_cotmatrix: manifold_meshes
 85/214 Test  #69: edge_flaps: verify ......................................   Passed    7.18 sec
        Start  93: intrinsic_delaunay_triangulation: two_triangles
 86/214 Test  #93: intrinsic_delaunay_triangulation: two_triangles .........   Passed    0.21 sec
        Start  94: intrinsic_delaunay_triangulation: skewed_grid
 87/214 Test  #94: intrinsic_delaunay_triangulation: skewed_grid ...........   Passed    0.20 sec
        Start  95: intrinsic_delaunay_triangulation: peaks
 88/214 Test  #95: intrinsic_delaunay_triangulation: peaks .................   Passed    0.20 sec
        Start  96: intrinsic_delaunay_triangulation: tet
 89/214 Test  #96: intrinsic_delaunay_triangulation: tet ...................   Passed    0.22 sec
        Start  97: is_delaunay: two_triangles
 90/214 Test  #97: is_delaunay: two_triangles ..............................   Passed    0.13 sec
        Start  98: is_edge_manifold: positive
 91/214 Test  #40: cr_vector_curvature_correction: annulus .................   Passed   14.72 sec
        Start  99: is_edge_manifold: negative
 92/214 Test  #99: is_edge_manifold: negative ..............................   Passed    0.74 sec
        Start 100: is_intrinsic_delaunay: two_triangles
 93/214 Test #100: is_intrinsic_delaunay: two_triangles ....................   Passed    0.35 sec
        Start 101: is_irregular_vertex: simple
 94/214 Test #101: is_irregular_vertex: simple .............................   Passed    0.62 sec
        Start 102: is_symmetric: sparse
 95/214 Test #102: is_symmetric: sparse ....................................   Passed    0.27 sec
        Start 103: is_symmetric: dense
 96/214 Test #103: is_symmetric: dense .....................................   Passed    0.29 sec
        Start 104: ismember: simple
 97/214 Test #104: ismember: simple ........................................   Passed    0.21 sec
        Start 105: isolines: broken-sphere
 98/214 Test #105: isolines: broken-sphere .................................   Passed    1.03 sec
        Start 106: iterative_closest_point: identity
 99/214 Test  #98: is_edge_manifold: positive ..............................   Passed    9.01 sec
        Start 107: knn
100/214 Test #107: knn .....................................................   Passed    0.22 sec
        Start 108: linprog: 2D-inequality
101/214 Test #108: linprog: 2D-inequality ..................................   Passed    0.21 sec
        Start 109: linprog: 2D-inequality+2-equality
102/214 Test #109: linprog: 2D-inequality+2-equality .......................   Passed    0.41 sec
        Start 110: linprog: 2D-inequality+1-equality
103/214 Test #110: linprog: 2D-inequality+1-equality .......................   Passed    0.14 sec
        Start 111: ListToMatrixTest: matrix
104/214 Test #111: ListToMatrixTest: matrix ................................   Passed    0.12 sec
        Start 112: lscm: lscm_energy_check
105/214 Test  #67: ears: two-boundary ......................................   Passed   20.95 sec
        Start 113: massmatrix: full
106/214 Test  #92: intrinsic_delaunay_cotmatrix: manifold_meshes ...........   Passed   15.85 sec
        Start 114: massmatrix: barycentric
107/214 Test #112: lscm: lscm_energy_check .................................   Passed    4.52 sec
        Start 115: massmatrix: cube_barycentric
108/214 Test #115: massmatrix: cube_barycentric ............................   Passed    0.21 sec
        Start 116: massmatrix: cube_full
109/214 Test #116: massmatrix: cube_full ...................................   Passed    0.33 sec
        Start 117: min_quad_with_fixed: dense
110/214 Test #117: min_quad_with_fixed: dense ..............................   Passed    0.22 sec
        Start 118: min_quad_with_fixed: Aeq
111/214 Test #118: min_quad_with_fixed: Aeq ................................   Passed    0.16 sec
        Start 119: moments: tet
112/214 Test #119: moments: tet ............................................   Passed    0.12 sec
        Start 120: orient_halfedges: sanity checks
113/214 Test  #12: bbw: decimated_knight ...................................   Passed   29.92 sec
        Start 121: igl_path_to_edges: basic_test
114/214 Test #121: igl_path_to_edges: basic_test ...........................   Passed    0.34 sec
        Start 122: igl_path_to_edges: loop_test
115/214 Test #122: igl_path_to_edges: loop_test ............................   Passed    0.25 sec
        Start 123: igl_path_to_edges: vector_basic_test
116/214 Test #123: igl_path_to_edges: vector_basic_test ....................   Passed    0.23 sec
        Start 124: igl_path_to_edges: vector_loop_test
117/214 Test #124: igl_path_to_edges: vector_loop_test .....................   Passed    0.25 sec
        Start 125: path_to_executable: example
118/214 Test #125: path_to_executable: example .............................   Passed    0.22 sec
        Start 126: pathinfo: examples
119/214 Test #126: pathinfo: examples ......................................   Passed    0.28 sec
        Start 127: per_face_normals: dot
120/214 Test #113: massmatrix: full ........................................   Passed    6.49 sec
        Start 128: polygon_corners: quads
121/214 Test #128: polygon_corners: quads ..................................   Passed    0.29 sec
        Start 129: principal_curvature: cylinder
122/214 Test #129: principal_curvature: cylinder ...........................   Passed    0.75 sec
        Start 130: procrustes
123/214 Test #130: procrustes ..............................................   Passed    0.26 sec
        Start 131: qslim: cylinder
124/214 Test #114: massmatrix: barycentric .................................   Passed    6.59 sec
        Start 132: quadprog: box3
125/214 Test #132: quadprog: box3 ..........................................   Passed    0.33 sec
        Start 133: quadprog: box2
126/214 Test #133: quadprog: box2 ..........................................   Passed    0.29 sec
        Start 134: random_points_on_mesh: decimated-knight
127/214 Test #134: random_points_on_mesh: decimated-knight .................   Passed    0.41 sec
        Start 135: random_points_on_mesh: minstd_rand0_reproduce
128/214 Test #131: qslim: cylinder .........................................   Passed    1.98 sec
        Start 136: random_points_on_mesh: minstd_rand_reproduce
129/214 Test #136: random_points_on_mesh: minstd_rand_reproduce ............   Passed    0.25 sec
        Start 137: random_points_on_mesh: mt19937_reproduce
130/214 Test #135: random_points_on_mesh: minstd_rand0_reproduce ...........   Passed    0.47 sec
        Start 138: random_points_on_mesh: mt19937_64_reproduce
131/214 Test   #4: AABB: dynamic ...........................................   Passed   35.76 sec
        Start 139: randperm: default_rng_reproduce_identity
132/214 Test #138: random_points_on_mesh: mt19937_64_reproduce .............   Passed    0.22 sec
        Start 140: randperm: minstd_rand0_reproduce_identity
133/214 Test #137: random_points_on_mesh: mt19937_reproduce ................   Passed    0.46 sec
        Start 141: randperm: minstd_rand_reproduce_identity
134/214 Test #139: randperm: default_rng_reproduce_identity ................   Passed    0.28 sec
        Start 142: randperm: mt19937_reproduce_identity
135/214 Test #140: randperm: minstd_rand0_reproduce_identity ...............   Passed    0.28 sec
        Start 143: randperm: mt19937_64_reproduce_identity
136/214 Test #141: randperm: minstd_rand_reproduce_identity ................   Passed    0.22 sec
        Start 144: randperm: default_identity
137/214 Test #142: randperm: mt19937_reproduce_identity ....................   Passed    0.21 sec
        Start 145: ray_mesh_intersect: one_triangle
138/214 Test #143: randperm: mt19937_64_reproduce_identity .................   Passed    0.30 sec
        Start 146: ray_mesh_intersect: corner-case
139/214 Test #145: ray_mesh_intersect: one_triangle ........................   Passed    0.23 sec
        Start 147: ray_mesh_intersect: corner-case2
140/214 Test #144: randperm: default_identity ..............................   Passed    0.30 sec
        Start 148: readDMAT: Comp
141/214 Test #146: ray_mesh_intersect: corner-case .........................   Passed    0.22 sec
        Start 149: readMESH: single-tet
142/214 Test #147: ray_mesh_intersect: corner-case2 ........................   Passed    0.21 sec
        Start 150: readMESH: no-triangles-line
143/214 Test #148: readDMAT: Comp ..........................................   Passed    0.25 sec
        Start 151: readMESH: mesh-version-formatted-2
144/214 Test #149: readMESH: single-tet ....................................   Passed    0.23 sec
        Start 152: readMSH
145/214 Test #150: readMESH: no-triangles-line .............................   Passed    0.21 sec
        Start 153: readOBJ: simple
146/214 Test #153: readOBJ: simple .........................................   Passed    0.16 sec
        Start 154: readOBJ: Obj with material
147/214 Test #151: readMESH: mesh-version-formatted-2 ......................   Passed    0.35 sec
        Start 155: readOFF: simple
148/214 Test #152: readMSH .................................................   Passed    0.47 sec
        Start 156: readPLY: cube_with_fold.ply
149/214 Test #154: readOBJ: Obj with material ..............................   Passed    0.25 sec
        Start 157: readPLY: bunny.ply
150/214 Test #155: readOFF: simple .........................................   Passed    0.30 sec
        Start 158: readPLY: mesh_error.ply
151/214 Test #156: readPLY: cube_with_fold.ply .............................   Passed    0.17 sec
        Start 159: readPLY: quad_cube.ply
152/214 Test #158: readPLY: mesh_error.ply .................................   Passed    0.24 sec
        Start 160: remesh_along_isoline: triangle_mesh
153/214 Test #159: readPLY: quad_cube.ply ..................................   Passed    0.26 sec
        Start 161: repmat: sparse rowMajor
154/214 Test #160: remesh_along_isoline: triangle_mesh .....................   Passed    0.17 sec
        Start 162: repmat: sparse colMajor
155/214 Test #127: per_face_normals: dot ...................................   Passed    6.25 sec
        Start 163: rigid_alignment: identity
156/214 Test #161: repmat: sparse rowMajor .................................   Passed    0.29 sec
        Start 164: seam_edges: tet
157/214 Test #162: repmat: sparse colMajor .................................   Passed    0.29 sec
        Start 165: segment_segment_intersect: examples
158/214 Test #163: rigid_alignment: identity ...............................   Passed    0.31 sec
        Start 166: setdiff: matrix
159/214 Test #164: seam_edges: tet .........................................   Passed    0.19 sec
        Start 167: slice: eigen-simple
160/214 Test #166: setdiff: matrix .........................................   Passed    0.18 sec
        Start 168: slice: eigen-random
161/214 Test #165: segment_segment_intersect: examples .....................   Passed    0.30 sec
        Start 169: slice: dense_identity
162/214 Test #167: slice: eigen-simple .....................................   Passed    0.22 sec
        Start 170: slice: sparse_identity
163/214 Test #168: slice: eigen-random .....................................   Passed    0.26 sec
        Start 171: slice: density_reverse
164/214 Test #170: slice: sparse_identity ..................................   Passed    0.20 sec
        Start 172: slice: random
165/214 Test #169: slice: dense_identity ...................................   Passed    0.29 sec
        Start 173: slice_into: eigen-random
166/214 Test #120: orient_halfedges: sanity checks .........................   Passed   10.01 sec
        Start 174: slice_into: dense_identity
167/214 Test #171: slice: density_reverse ..................................   Passed    0.22 sec
        Start 175: slice_into: density_reverse
168/214 Test #172: slice: random ...........................................   Passed    0.21 sec
        Start 176: slice_into: sparse_identity
169/214 Test #173: slice_into: eigen-random ................................   Passed    0.26 sec
        Start 177: slice_into: every-other
170/214 Test #174: slice_into: dense_identity ..............................   Passed    0.26 sec
171/214 Test #175: slice_into: density_reverse .............................   Passed    0.24 sec
        Start 178: slice_mask/find: random
        Start 179: slice_sorted: correctness
172/214 Test #176: slice_into: sparse_identity .............................   Passed    0.28 sec
        Start 180: SortTest: random
173/214 Test #177: slice_into: every-other .................................   Passed    0.27 sec
        Start 181: sparse_voxel_grid: unique
174/214 Test #180: SortTest: random ........................................   Passed    0.21 sec
        Start 182: split_nonmanifold: edge-fan
175/214 Test #178: slice_mask/find: random .................................   Passed    0.31 sec
        Start 183: split_nonmanifold: vertex-boundary
176/214 Test #181: sparse_voxel_grid: unique ...............................   Passed    0.26 sec
        Start 184: split_nonmanifold: edge-disk-flap
177/214 Test #157: readPLY: bunny.ply ......................................   Passed    2.22 sec
        Start 185: split_nonmanifold: edge-disk-tent
178/214 Test #183: split_nonmanifold: vertex-boundary ......................   Passed    0.19 sec
        Start 186: split_nonmanifold: vertex-kiss
179/214 Test #182: split_nonmanifold: edge-fan .............................   Passed    0.25 sec
        Start 187: split_nonmanifold: non-orientable
180/214 Test #179: slice_sorted: correctness ...............................   Passed    0.53 sec
        Start 188: split_nonmanifold: flap
181/214 Test #185: split_nonmanifold: edge-disk-tent .......................   Passed    0.19 sec
        Start 189: split_nonmanifold: crosses
182/214 Test #187: split_nonmanifold: non-orientable .......................   Passed    0.19 sec
        Start 190: squared_edge_lengths: cube
183/214 Test #184: split_nonmanifold: edge-disk-flap .......................   Passed    0.32 sec
        Start 191: super_fibonacci: simple
184/214 Test #186: split_nonmanifold: vertex-kiss ..........................   Passed    0.29 sec
        Start 192: tet_tet_adjacency: dot
185/214 Test #188: split_nonmanifold: flap .................................   Passed    0.27 sec
        Start 193: tri_tri_intersection_test_3d intersect
186/214 Test #191: super_fibonacci: simple .................................   Passed    0.21 sec
        Start 194: tri_tri_intersection_test_3d not intersect
187/214 Test #190: squared_edge_lengths: cube ..............................   Passed    0.34 sec
        Start 195: tri_tri_intersection_test_3d coplanar
188/214 Test #193: tri_tri_intersection_test_3d intersect ..................   Passed    0.33 sec
        Start 196: triangle_triangle_adjacency: dot
189/214 Test #192: tet_tet_adjacency: dot ..................................   Passed    0.38 sec
190/214 Test #194: tri_tri_intersection_test_3d not intersect ..............   Passed    0.22 sec
        Start 197: triangle_triangle_intersect: shared-edge
        Start 198: triangulated_grid: area
191/214 Test #195: tri_tri_intersection_test_3d coplanar ...................   Passed    0.26 sec
        Start 199: turning_number: pentagon
192/214 Test #189: split_nonmanifold: crosses ..............................   Passed    0.70 sec
        Start 200: turning_number: heptagram
193/214 Test #198: triangulated_grid: area .................................   Passed    0.18 sec
        Start 201: unique: matrix
194/214 Test #197: triangle_triangle_intersect: shared-edge ................   Passed    0.25 sec
        Start 202: unique_rows: matrix
195/214 Test #201: unique: matrix ..........................................   Passed    0.13 sec
        Start 203: igl_unique_simplices: duplicate_triangles
196/214 Test #199: turning_number: pentagon ................................   Passed    0.21 sec
        Start 204: upsample: single_triangle
197/214 Test #202: unique_rows: matrix .....................................   Passed    0.14 sec
        Start 205: upsample: V_comes_first_F_ordering
198/214 Test #200: turning_number: heptagram ...............................   Passed    0.29 sec
        Start 206: voronoi_mass: equilateral-tetrahedra
199/214 Test #204: upsample: single_triangle ...............................   Passed    0.17 sec
        Start 207: voronoi_mass: right-tetrahedra
200/214 Test #203: igl_unique_simplices: duplicate_triangles ...............   Passed    0.24 sec
        Start 208: voronoi_mass: obtuse-tetrahedra
201/214 Test #206: voronoi_mass: equilateral-tetrahedra ....................   Passed    0.20 sec
        Start 209: winding_number: bunny
202/214 Test #207: voronoi_mass: right-tetrahedra ..........................   Passed    0.16 sec
        Start 210: writeMSH
203/214 Test #208: voronoi_mass: obtuse-tetrahedra .........................   Passed    0.17 sec
        Start 211: writeOFF: quads
204/214 Test #211: writeOFF: quads .........................................   Passed    0.13 sec
        Start 212: writePLY: bunny.ply
205/214 Test #106: iterative_closest_point: identity .......................   Passed   22.78 sec
        Start 213: writePLY: bunny.ply float
206/214 Test #210: writeMSH ................................................   Passed    1.14 sec
        Start 214: eigs: grid
207/214 Test #214: eigs: grid ..............................................   Passed    0.91 sec
208/214 Test #212: writePLY: bunny.ply .....................................   Passed    2.85 sec
209/214 Test #213: writePLY: bunny.ply float ...............................   Passed    2.53 sec
210/214 Test #196: triangle_triangle_adjacency: dot ........................   Passed    6.56 sec
211/214 Test #205: upsample: V_comes_first_F_ordering ......................   Passed    6.29 sec
212/214 Test  #80: fast_winding_number: meshes .............................   Passed   63.46 sec
213/214 Test #209: winding_number: bunny ...................................   Passed   31.18 sec
214/214 Test  #31: cotmatrix: poly .........................................   Passed   72.03 sec

100% tests passed, 0 tests failed out of 214

Total Test time (real) =  73.01 sec
```

### CODE REVIEW

File: projects/libigl/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+cd build-dir
+# TODO: Skipping some failing tests for now.
```
Comment (vitaliset) [inline projects/libigl/run_tests.sh:18]:
```
The following tests FAILED:
          5 - AABB: intersect (Timeout)
         18 - boundary_facets: single_tri (Subprocess aborted)
         92 - increment_ulp: dense (Timeout)
        171 - signed_distance: single_tet (Timeout)
        172 - signed_distance: single_triangle (Failed)
        173 - signed_distance: dimension-templates (Failed)
```

**Review by DavidKorczynski — APPROVED** [review]
